man/systemd-sbsign.xml

```diff
@@ -91,9 +91,9 @@
 
         <listitem><para>When this option is specified, the <command>sign</command> command writes the data
         that should be signed to the path specified with <option>--output=</option> instead of writing the
-        signed PE binary. This data can then be signed out of band after which the signature can be attached
-        to the PE binary using the <option>--signed-data=</option> and
-        <option>--signed-data-signature=</option> options.</para>
+        signed output. This data can then be signed out of band after which the signature can be attached
+        using the <option>--signed-data=</option> and <option>--signed-data-signature=</option> options.
+        </para>
```

Comment:
as above, i think docs should be helpful, please enumerate what this outputs, do not make this too generic to be understandable.